yb-voyager/src/ybversion/constants.go

```diff
@@ -58,9 +58,9 @@ func init() {
 		panic("could not create version 2024.2.1.0")
 	}
 
-	V2024_2_2_2, err = NewYBVersion("2024.2.2.2")
+	V2024_2_2_2, err = NewYBVersion("2024.2.2.3")
```

Comment:
Create a new variable `V2024_2_2_3` and update the `LatestStable` to point to this one.

Reply:
refer this PR #2569

Reply:
Thanks!

Reply:
I have merged this, you can just rebase main now.